<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>807</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OneDriveGUI</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../OneDriveGUI_POC_recovered-multi/icons8-clouds-48.png</normaloff>../../../OneDriveGUI_POC_recovered-multi/icons8-clouds-48.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::DefaultContextMenu</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 120, 212);
color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="pushButton_new_profile">
          <property name="maximumSize">
           <size>
            <width>9999</width>
            <height>9999</height>
           </size>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">alignment=Qt.AlignCenter
</string>
          </property>
          <property name="text">
           <string>Create/ Import OneDrive profile</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>420</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAdvanced">
    <property name="title">
     <string>Advanced</string>
    </property>
    <widget class="QMenu" name="menuDebug">
     <property name="title">
      <string>Debug</string>
     </property>
     <widget class="QMenu" name="menuService">
      <property name="title">
       <string>Service</string>
      </property>
      <addaction name="actionStart_Service"/>
      <addaction name="actionStop_Service"/>
      <addaction name="actionRestart_Service"/>
     </widget>
     <widget class="QMenu" name="menuMonitor">
      <property name="title">
       <string>Monitor</string>
      </property>
      <addaction name="actionStart_Monitor"/>
      <addaction name="actionStop_Monitor"/>
      <addaction name="actionForce_Resync"/>
     </widget>
     <widget class="QMenu" name="menuStatus">
      <property name="title">
       <string>Status</string>
      </property>
      <addaction name="actionRefresh_Service_Status"/>
      <addaction name="actionObtain_Sync_Status"/>
     </widget>
     <addaction name="menuService"/>
     <addaction name="menuMonitor"/>
     <addaction name="menuStatus"/>
    </widget>
    <widget class="QMenu" name="menuaccount2">
     <property name="title">
      <string>account2</string>
     </property>
     <addaction name="actionstart"/>
    </widget>
    <addaction name="menuDebug"/>
    <addaction name="menuaccount2"/>
   </widget>
   <addaction name="menuAdvanced"/>
  </widget>
  <action name="actionfghgfh">
   <property name="text">
    <string>fghgfh</string>
   </property>
  </action>
  <action name="actionStart_Service">
   <property name="text">
    <string>Start Service</string>
   </property>
  </action>
  <action name="actionStop_Service">
   <property name="text">
    <string>Stop Service</string>
   </property>
  </action>
  <action name="actionRestart_Service">
   <property name="text">
    <string>Restart Service</string>
   </property>
  </action>
  <action name="actionStart_Monitor">
   <property name="text">
    <string>Start Monitor</string>
   </property>
  </action>
  <action name="actionStop_Monitor">
   <property name="text">
    <string>Stop Monitor</string>
   </property>
  </action>
  <action name="actionForce_Resync">
   <property name="text">
    <string>Force Resync</string>
   </property>
  </action>
  <action name="actionRefresh_Service_Status">
   <property name="text">
    <string>Refresh Service Status</string>
   </property>
  </action>
  <action name="actionObtain_Sync_Status">
   <property name="text">
    <string>Obtain Sync Status</string>
   </property>
  </action>
  <action name="actionTest_Service">
   <property name="text">
    <string>Test Service</string>
   </property>
  </action>
  <action name="actionstart">
   <property name="text">
    <string>start</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
